02-Design/01-Principles of Security Engineering.md

```diff
@@ -18,22 +18,22 @@ Limits the damage in case of exploited vulnerability.
 In order to apply this principle, proper granularity of privileges and permissions should be established.
 
 ## Separation of Duties
-Also known as the [compartmentalization principle][1], or separation of privilege, separation of duties is a security principle which states that the successful completion of a single task is dependent upon two or more conditions that need to be met and just one of the conditions will be insufficient in completing the task by itself.
+Also known as the [compartmentalization principle][1], or separation of privilege, separation of duties is a security principle which states that the successful completion of a single task is dependent upon two or more conditions that are insufficient for completing the task by themselves.
```

Comment:
I like most of it, but completion of a single task doesn't agree with the plural of themselves.

Reply:
True, changed back to itself and pushed